tests/python/unittest/test_auto_scheduler_measure.py

```diff
@@ -18,14 +18,51 @@
 """ Test measurement and log serialization. """
 
 import tvm
-from tvm import auto_scheduler
+import topi
+from tvm import te, auto_scheduler
 import tempfile
 
 from test_auto_scheduler_common import get_tiled_matmul
 
 
 def test_record():
-    dag, s = get_tiled_matmul()
+    A = te.placeholder((512, 512), name='A')
```

Comment:
Move the checker in L67 to here. If LLVM is disabled, we don't have to even build this DAG.